scripts/test.sh

```diff
@@ -8,23 +8,27 @@ JENKINS_CONF=jenkins.py.dist
 
 ENV=env
 
-PYTHONPATH="."
+PYTHONPATH='.'
```

Comment:
That's debugging cruft. There's no need for the quotes, so I'll just remove them when I add comments about deprecation.